### PR TITLE
OCM-13019 | feat: Describe Cluster changes for hcpsharedvpc

### PR DESCRIPTION
**Pull request by hunterkepley**

### CODE REVIEW

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -447,13 +447,19 @@ func run(cmd *cobra.Command, argv []string) {
 			cluster.AWS().STS().OIDCEndpointURL(), managementType)
 	}
 	if cluster.AWS().PrivateHostedZoneID() != "" {
-		str = fmt.Sprintf("%s"+"Private Hosted Zone:\n", str)
+		str = fmt.Sprintf("%s"+"Shared VPC Config:\n", str)
 		str = fmt.Sprintf("%s"+
-			" - ID:                      %s\n", str,
+			" - Ingress Private HZ ID:   %s\n", str,
 			cluster.AWS().PrivateHostedZoneID())
 		str = fmt.Sprintf("%s"+
```
Comment (gdbranco) [inline cmd/describe/cluster/cmd.go:454]: should check if it is hypershift I think

Comment (gdbranco) [reply cmd/describe/cluster/cmd.go:454]: But we still need to provide the old params for non hypershift :0

Comment (hunterkepley) [reply cmd/describe/cluster/cmd.go:454]: oh sorry, got confused from the requirements (**replace** X with Y) - will put back for non HS

**Pull request merged**